### ISSUE TITLE
Cannot check for property existence. 

### ISSUE THREAD

**Issue by matthewarkin** [balderdashy/sails-disk#37]
Using sails-mongo I can do a query like:

````
Model.find({
        property: {'!': undefined},
})
````

to get an array of Models where property is set, in sails-disk this only returns an empty array.

**Comment by jvanalst** [comment]
I have a similar related issue. I'm trying to check for a field that is null, but it doesn't work with the Sails-Disk adaptor.

```javascript
Model.find({
  where:  {
    field: null
  }
});
```

Because the field is not required it never gets set, and thus is missing entirely under Sails-Disk. Whereas under a MySQL schema the field would have to be null by default.

**Comment by sailsbot** [comment]
Thanks for posting, @matthewarkin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
